localstripe/resources.py

```diff
@@ -2058,12 +2060,15 @@ def __init__(self, customer=None, metadata=None, items=None,
         self.status = 'incomplete'
         self.trial_end = None
         self.trial_start = None
+        self.trial_period_days = trial_period_days
```

Comment:
If `trial_period_days` is provided, it needs to be checked to be an integer. See an example here for `Charge`:
https://github.com/adrienverge/localstripe/blob/3d0110ac0df320b24ffe275a5ba3645a97eef2e6/localstripe/resources.py#L328